clinguin_ui/ui_menu_bar.lp

```diff
@@ -0,0 +1,183 @@
+#const help_message = @concat(
+    "<p><i>This UI allows to create a configuration</i></p>",
+    "<h5>Add objects</h5>",
+    "<p>In the add 'Add object' section one can add a new object to the system. Only leaf classes can be added as objects. Click on + to add the </p>",
+    "<h5>Edit objects</h5>",
+    "<p>Click on the node in the configuration graph to edit the object. The lower right pannel will show the selected node and posible changes. </p>",
+    "<h5>Partial and full configuration</h5>",
+    "<p>The yellow button on the menu can toggle between partial and full configuration. In partial configuration objects might not have values for attributes and lowebounds might not be filled. In full configuration the system will enforce the complete configuration. </p>",
```

Comment:
Please check all text elements for grammar and spelling, I don't want to deliver all the details on this ...